tool/src/test/java/org/wildfly/security/tool/VaultCommandTest.java

```diff
@@ -69,12 +121,24 @@ public void singleConversionBasicTest() throws Exception {
      */
     @Test
     public void bulkConversionBasicTest() throws Exception {
-        String[] args = {"--bulk-convert", "target/test-classes/bulk-vault-conversion-desc"};
+        String[] args;
+        if(IS_IBM) {
+            args = new String[]{"--bulk-convert", "target/test-classes/bulk-vault-conversion-desc-ibm"};
+        }
+        else {
+            args = new String[]{"--bulk-convert", "target/test-classes/bulk-vault-conversion-desc"};
+        }
         // conversion
         String output = executeCommandAndCheckStatusAndGetOutput(args);
         String[] parts = output.split("converted to credential store");
         Assert.assertTrue("Three credential stores has to be created", parts.length == 4);
-        Assert.assertTrue("Check file names must pass", output.indexOf("vault-v1/vault-jceks.keystore") > 0 && output.indexOf("vault-v1-more/vault-jceks.keystore") > 0);
+        if(IS_IBM) {
+            Assert.assertTrue("Check file names must pass", output.indexOf("vault-v1/vault-jceks-ibm.keystore") > 0 && output.indexOf("vault-v1-more/vault-jceks-ibm.keystore") > 0);
```

Comment:
I think it would be good to update the readme file with the commands used to generate the test files for IBM JDK. It would also be good to reference ELY-1792 in the readme as well.

Reply:
Done